packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -51,6 +53,7 @@ import {
   DiagnosticChangedEventListner,
   CallAdapterCallEndedEvent
 } from './CallAdapter';
+import { CallAdapterOptionalFeatures } from './CallAdapter';
```

Comment:
Should this be CC? 
CallAdapterOptionalFeatures type is a @beta feature.

Reply:
After speaking with Don privately, he clarified that as long as the Stable API is not impacted it is fine not to CC internally.